data/ActType.csv

```diff
@@ -1,6 +1,9 @@
 action_ID,action_name,created,created_by,last_modified,last_modified_by
 AT01,to read,2017-07-12,OS,2017-07-12,OS
-AT02,to copy,2017-07-12,OS,2017-07-12,OS
+AT02,to copy (shouchao),2017-07-12,OS,2017-07-28,LH
```

Comment:
Yes, I thought of that too, but shouchao so neatly captures more than copy. And as I am writing a book on shouchao practices, I'd insist even though it is not for purists.